scripts/img_check.sh

```diff
@@ -161,8 +161,15 @@ function checkRoot {
                         do
                              if  [ "${key}" == "${uhome}/.ssh/authorized_keys" ]; then
                             
-                                if [ "$( cat "${key}" | wc -c)" -gt 50 ]; then
-                                    echo -en "\e[41m[FAIL]\e[0m User \e[1m${user}\e[0m has a populated authorized_keys file in \e[93m${key}\e[0m\n"
+                                if [ "$( cat "${key}" | wc -c )" -gt 50 ]; then
+				    linesinauthkeysfile=$( cat ${key} | grep ssh | wc -l )	
```

Comment:
Super small knit pick on the line spacing proceeding `linesinauthkeysfile` needs a few more spaces to be inline with above code blocks.